dozer-types/src/types/field.rs

```diff
@@ -14,7 +14,9 @@ pub const DATE_FORMAT: &str = "%Y-%m-%d";
 #[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, PartialOrd, Ord, Hash)]
 pub enum Field {
     UInt(u64),
+    UInt128(u128),
```

Comment:
let's wait to introduce these. There is going to be a lot of work to extend all math operators. Let's do it as a second step

Reply:
+ agree